src/app/codegen-data-model-provider/CodegenDataModelProvider.h

```diff
@@ -216,16 +229,15 @@ class CodegenDataModelProvider : public chip::app::DataModel::Provider
     const EmberAfCluster * FindServerCluster(const ConcreteClusterPath & path);
 
     /// Find the index of the given attribute id
-    std::optional<unsigned> TryFindAttributeIndex(const EmberAfCluster * cluster, chip::AttributeId id) const;
```

Comment:
The cleanup of existing code should really have been in a separate PR.  This makes it very hard to tell what's substantive changes and what's cleanup that is not expected to have no effect.

Reply:
https://github.com/project-chip/connectedhomeip/pull/36737